databases/week2/hyf-task.sql

```diff
@@ -0,0 +1,75 @@
+-- Part1
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+    task(title, description, created, updated, due_date, status_id, user_id)
+VALUES
+    ('Bake muffins', 'For 5 people', '2020-10-04 03:16:42', '2020-10-05 16:56:58', '2020-11-05 16:50:50', 2, 6);
+    
+-- Change the title of a task
+UPDATE
+    task
```

Comment:
Would suggest to keep the update [table name] on the same line.

Pointer: improves code readability and clarity

Reply:
also update other queries where this happens.

Reply:
Thank you for your review!